bundles/Billing/Service/BillingService.php

```diff
@@ -835,6 +838,15 @@ public function getProjectWorklogsWithMetadata($projectId)
 
             $issue->versions = $issueVersions;
 
+            if (isset($issue->accountKey)) {
```

Comment:
A comment about what's going on here would be nice, e.g. “Enrich issues with account name”.